frontend-ssr/redux/exercise-submissions/types.ts

```diff
@@ -0,0 +1,21 @@
+export interface Submission {
+    username: string,
+    avatarSrc: string,
+    id: string,
+    chapter: ChapterType,
+    time: number,
+    type: SubmissionType,
+    editing?: boolean,
+}
+export interface SubmissionState {
+    submissions: Submission[],
+}
+export enum ChapterType {
+    HTML = 'html',
```

Comment:
This enum should be in a generic `constants.ts` file. I have a feeling we'll use it in multiple parts of the app.